docs/crio.8.md

```diff
@@ -360,15 +360,15 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--registry**="": Registry to be prepended when pulling unqualified images. Can be specified multiple times.
 
-**--root, -r**="": The CRI-O root directory. (default: /var/lib/containers/storage)
+**--root, -r**="": The CRI-O root directory. (default: /var/home/pehunt/.local/share/containers/storage)
```

Comment:
`s/pehunt/user/g?`

Reply:
It should be still point to `/var/lib/containers/storage`. :slightly_smiling_face: